subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/results/ResultsStore.java

```diff
@@ -34,4 +35,6 @@
      * Returns the n most recent instances of the given test which are younger than the max age.
      */
     PerformanceTestHistory getTestResults(PerformanceExperiment experiment, int mostRecentN, int maxDaysOld, String channel);
+
+    Map<PerformanceExperiment, Long> getEstimatedExperimentTimes(OperatingSystem operatingSystem);
```

Comment:
I would name it `getEstimatedExperimentTimesMillis` or `getEstimatedExperimentTimesSecond` to clearly indicate what's inside.

Reply:
Fixed in https://github.com/gradle/gradle/pull/14672